superdesk/io/commands/update_ingest.py

```diff
@@ -586,6 +586,9 @@ def ingest_item(item, provider, feeding_service, rule_set=None, routing_scheme=N
                 f"updating items is disabled on provider '{provider['name']}'."
             )
             return False, []
+        elif old_item and old_item.get("version_creator"):
```

Comment:
I wonder if this would somehow affect the articles as well, might be good to call here the respective service and keep the logic there so we can handle it specifically for events, eg.:
```py
elif old_item and not ingest_service.should_update(old_item, item, provider):
```
and have this `old_item.get("version_creator")` in events ingest service
and add some default implementation in the core `IngestService`:
```py
def should_update(self, old_item, new_item, provider):
    return True
```